src/OSPSuite.SimModel/Simulation.cs

```diff
@@ -185,7 +185,7 @@ internal class SimulationImports
       public static extern IntPtr ExportSimulationToMatlabCode(IntPtr simulation, string outputFolder, bool fullMode, out bool success, out string errorMessage);
 
       [DllImport(SimModelImportDefinitions.NATIVE_DLL, CallingConvention = SimModelImportDefinitions.CALLING_CONVENTION)]
-      public static extern IntPtr ExportSimulationToCppCode(IntPtr simulation, string outputFolder, bool fullMode, out bool success, out string errorMessage);
+      public static extern IntPtr ExportSimulationToCppCode(IntPtr simulation, string outputFolder, bool fullMode, string modelName, out bool success, out string errorMessage);
```

Comment:
Enables giving a model name (otherwise every models is exported as "Standard.cpp"